File: Practice(Beginner)/Python3/Second_Largest.py
# Source: https://www.codechef.com/problems/FLOW017

n = int(input())
for it in range(n):
    a,b,c = map(int,input().split())
    if(b < a < c or c < a < b):
        print(a)
    elif(a < b < c or c < b < a):
        print(b)
    else:
        print(c)

File: Practice(Beginner)/C++/Second Largest.cpp
// Source: https://www.codechef.com/problems/FLOW017

#include <bits/stdc++.h>
using namespace std;

int main() {

    int t, a, b, c;

    cin >> t;
    for (int i = 0; i < t; ++i) {
        cin >> a >> b >> c;
        if((b > a and a > c) or (c > a and a > b)){
            cout << a << endl;
        }else if((a > b and b > c) or (c > b and b > a)){
            cout << b << endl;
        }else{
            cout << c << endl;
        }
    }

    return 0;
}
